### DISCUSSION TITLE
Know if Hi.Evetns supports certain features

### DISCUSSION THREAD

**Discussion by WaloGomez** [HiEventsDev/Hi.Events#280] [General]
Hello everyone,
Congratulations for your great work !

I am evaluating using Hi.Events. I wanted to know if he supports:

1. Creation of concurrent events in a period of dates. For example, a guided museum tour that takes place every day of a week, at the same times each day. Being able to reserve by date/time, updating the stock in the same way with the purchase of a ticket.
2. Create an event with a type of audience for example (Adults 3 units, Retirees 2 units) and that the ticket can be purchased in the same order and that the 5 units are discounted to the maximum, assigning the 
event.
It is possible via rest api.
1. Get all active events (description and media (video, images, etc)).
2. Get all the dates on which you can reserve and for each date the available hours.
3. Create an order and change the status of an event.
4. Use your own checkout and then update the order's payment details.

Thank you so much
Walter

**Comment by islandmix** [comment]
Might need more context on your questions... i am not a developer/contributor but here are my 2 cents

1. The default app in its current format is more appropriate for night club event ticketing. Your scenario is more like conference event ticketing or a booking system.
2. What is "units"? is that quantity? You can create event tickets and call them Adult/Senior with capacity settings
3. Is this a report you are requesting?
4. See response #1
5. Is the "order" the ticket? If you cancel the event then I am not sure what happens
6. That is an offline enhancement that is on the roadmap... but not yet implemented.
